evento/src/views/MyEvents/index.js

```diff
@@ -37,9 +43,19 @@ class MyEvents extends Component {
 			return <h4>{this.state.errorMessage}</h4>
 		}
 
+		let upcomingHeader = "You are attending to the following events";
```

Comment:
I'd say that move this code to 'getUpcomingEventsHeader' and 'getPastEventsHeader' functions to move bloat away from the render.